### PR TITLE
MARP-452 check accessibility of new website

### PR DESCRIPTION
**Pull request by tutn-axonivy**

### CODE REVIEW

**Review by ndkhanh-axonivy — COMMENTED** [review]
Please check this comments, please pull code from develop and handle conflicts. Thanks!

File: marketplace-ui/src/app/shared/components/header/search-bar/search-bar.component.html [hunk 1]
```diff
@@ -4,33 +4,27 @@
...
-          </span>
+        <span class="input-group-text text-primary h-100 border-0 bg-secondary">
+          <i class="bi bi-search"></i>
+        </span>
       </div>
-      <input
-        class="form-control bg-secondary me-2 border-0 header__search-input bg-secondary"
-        type="input"
-        [placeholder]="
+      <input autocomplete="on" [lang]="languageService.selectedLanguage()" class="form-control bg-secondary me-2 border-0 header__search-input bg-secondary"
```
[the remark this answers is not in the document]
Comment (tutn-axonivy) [reply marketplace-ui/src/app/shared/components/header/search-bar/search-bar.component.html:11]: Its 1.3.5 Identify input purpose (AA)

File: marketplace-ui/src/app/modules/product/product-card/product-card.component.html [hunk 2]
```diff
@@ -7,7 +7,10 @@
...
       height="70"
       [ngSrc]="product | logo"
-      [alt]="product.names | multilingualism: languageService.selectedLanguage()" />
+      [alt]="
+        product.names | multilingualism: languageService.selectedLanguage()
+      "
+      [lang]="languageService.selectedLanguage()" />
     @if (isShowInRESTClientEditor) {
       <div
         class="card__tag lh-md px-2 py-1 text-capitalize"
```
[the remark this answers is not in the document]
Comment (tutn-axonivy) [reply marketplace-ui/src/app/modules/product/product-card/product-card.component.html:16]: It does not use translate or multilingualism pipe so it should not include lang

**Review by ndkhanh-axonivy — APPROVED** [review]
LGTM

**Pull request merged**